### PR TITLE
Updated XCM-Transactor after refactoring

### PR DESCRIPTION
**Pull request by albertov19**
### Description

This PR updates the XCM-transactor docs to include the revamp introduced in https://github.com/PureStake/moonbeam/pull/1725 

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [x] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [x] No additional PRs are required after the translations are done

### CODE REVIEW

File: builders/xcm/xcm-transactor.md
Comment (eshaben) [inline builders/xcm/xcm-transactor.md:139]:
```suggestion
10. Enter the inner call that will be executed in the destination chain. This is the encoded call data of the pallet, method, and input values to be called. It can be constructed in Polkadot.js Apps (must be connected to the destination chain), or using the [Polkadot.js API](/builders/build/substrate-api/polkadot-js-api/){target=_blank}. For this example, the inner call is `0x04000030fcfb53304c429689c8f94ead291272333e16d77a2560717f3a7a410be9b208070010a5d4e8`, which is a simple balance transfer of 1 `UNIT` to Alice's account in the relay chain. You can decode the call in [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffrag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network#/extrinsics/decode){target=_blank}
```

File: builders/xcm/xcm-transactor.md
Comment (eshaben) [inline builders/xcm/xcm-transactor.md:208]:
```suggestion
9. Enter the inner call that will be executed in the destination chain. This is the encoded call data of the pallet, method, and input values to be called. It can be constructed in Polkadot.js Apps (must be connected to the destination chain), or using the [Polkadot.js API](/builders/build/substrate-api/polkadot-js-api/){target=_blank}. For this example, the inner call is `0x030044236223ab4291b93eed10e4b511b37a398dee5513000064a7b3b6e00d`, which is a simple balance transfer of 1 token of the destination chain to Alice's account there
```

**Review by eshaben — APPROVED** [review]
LGTM 🚀

**Comment by eshaben** [comment]
Will wait for Gorka to review before merging

File: builders/xcm/xcm-transactor.md
Comment (girazoki) [inline builders/xcm/xcm-transactor.md:72]: fee_amount is missing

Comment (girazoki) [reply builders/xcm/xcm-transactor.md:72]: oh sorry, just saw it

**Comment by girazoki** [comment]
Is there anywhere we can indicate the user that the previous precompile interface (V1) is still live at a certain address (2054)? Because the new precompile interface (V2) has only been installed in moonbase alpha for now (at address 2061)

**Comment by albertov19** [comment]
> Is there anywhere we can indicate the user that the previous precompile interface (V1) is still live at a certain address (2054)? Because the new precompile interface (V2) has only been installed in moonbase alpha for now (at address 2061)

Good point! I've added a part with V1 and V2 addresses

**Review by girazoki — APPROVED** [review]

**Pull request merged**
